cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1101,7 +1100,7 @@ SwiftGenerator::writeMembers(
                     out,
                     type,
                     member->defaultValueType(),
-                    defaultValue,
+                    member->defaultValue().value_or(""), // TODO: revisit this code
```

Comment:
Most of the code in this function seems to be about auto-escaping to avoid collision ... and we're reworking this code already.